data/filter/FieldFilter.ts

```diff
@@ -128,14 +130,14 @@ export class FieldFilter extends Filter {
         switch (op) {
```

Comment:
would `rValue` and `fValue` be more helpful here?  Very hard to keep track of which is which.